ansible/roles_ocp_workloads/ocp4_workload_example_dedicated_cluster/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+become_override: False
+ocp_username: wkulhane-redhat.com
+silent: False
+
+# If your workload needs customization options provide variables to be used.
+# Variable names must be prefixed by the role name "<role name>_<variable>".
+# So because this example workload is called "ocp4_workload_example" the variables
+# should be named "ocp4_workload_example_variable".
+#
+# You can override the defaults as parameters to the Ansible run that runs
+# your workload.
+
+ocp4_workload_example_variable_1: "value 1"
```

Comment:
Variable names are wrong. Should be ocp4_workload_example_dedicated_cluster_<variable>

Reply:
resolved throughout